config.json

```diff
@@ -432,7 +432,12 @@
       "core": false,
       "difficulty": 5,
       "slug": "run-length-encoding",
-      "topics": null,
+      "topics": [
+        "strings",
+        "integers",
+        "pattern_matching",
+	"transforming"
```

Comment:
A small indentation issue here. `transforming` should be indented two spaces afer `topics`, same as the other topics :)